odalic/src/main/java/cz/cuni/mff/xrg/odalic/outputs/annotatedtable/DefaultResultToAnnotatedTableAdapter.java

```diff
@@ -57,24 +61,24 @@ public AnnotatedTable toAnnotatedTable(Result result, Input input, Configuration
     return new AnnotatedTable(input.identifier(), new TableSchema(columns));
   }
   
-  private TableColumn createOriginalColumn(String columnName) {
+  private TableColumn createOriginalColumn(String columnName, boolean chosenIsEmpty) {
```

Comment:
Introducing flags in method parameters, such as the choseIsEmpty, is a bad habit. Provide two methods instead (albeit with similar code): one for creating chosen original column, one for creating regular one). It will greatly simplify the code and helps you to get rid off those pesky ternary operators.